Breadcrumbs.py

```diff
@@ -108,4 +141,223 @@ def get_points_by_row(row):
               breadcrumbs[index_of_breadcrumb] = breadcrumbs[index_of_breadcrumb][0:length_of_trim]
           break
       number_of_characters_left -= current_length
-    view.set_status('breadcrumbs',separator.join(breadcrumbs+['']))
+  return breadcrumbs
+
+
+def copy(view, text):
+  sublime.set_clipboard(text)
+  sublime.status_message('Breadcrumbs copied to clipboard')
+
+
+if viewevents_available is not True:
+  class BreadcrumbsEventListenerST2(sublime_plugin.EventListener):
+
+    def on_selection_modified(self, view):
+      if get_statusbar_enabled(view.settings()):
+        current_row = view.rowcol(view.sel()[0].b)[0]
+        breadcrumbs = make_breadcrumbs(view, current_row, shorten=True)
+
+        if len(breadcrumbs) > 0:
+          view.set_status('breadcrumbs', get_separator(view).join(breadcrumbs))
+      else:
+        view.erase_status('breadcrumbs')
+
+
+class BreadcrumbsEventListenerST3(sublime_plugin.ViewEventListener):
```

Comment:
I insist that you wrap this thing in an if (or rather an "else:") . It is not enough that you simply have `is_applicable` defined, as in ST2 there is no such thing `sublime_plugin.ViewEventListener` and you simply can not extend this class, and thus can not even define this BreadcrumbsEventListenerST3 class without throwing an error, which causes all the subsequent code to be ignored. The particular error is:
```
Traceback (most recent call last):
  File ".\sublime_plugin.py", line 62, in reload_plugin
  File ".\Breadcrumbs.py", line 166, in <module>
    class BreadcrumbsEventListenerST3(sublime_plugin.ViewEventListener):
AttributeError: 'module' object has no attribute 'ViewEventListener'
```